libcore/src/IO/Trajectories.h

```diff
@@ -54,6 +57,8 @@ class Trajectories
         _optionalOutputOptions = options;
     }
 
+    virtual void SetPrecision(unsigned int precision) { _precision = precision; }
```

Comment:
Is there a use case that the precision changes during simulation? If not, it might be better to add the constructor, to avoid any misuse

Reply:
No, precision should not change during simulation. It's just for output reasons